src/SourceBuild/content/eng/tools/BuildComparer/Program.cs

```diff
@@ -373,31 +379,41 @@ public async Task EvaluatePackageContents(AssetMapping mapping)
         }
     }
 
+    /// <summary>
+    /// Compare the file lists of packages, identifying missing and extra files.
+    /// </summary>
+    /// <param name="mapping"></param>
```

Comment:
Nit, what value do empty doc tags provide?  It bloats the code and erodes the value of the actual documentation.

Reply:
Mind if I fix this in the next iteration? There are some more tests to add to this.

Reply:
That's fine with me.